README.md

```diff
@@ -1,71 +1,81 @@
 # openpilot/etc. on Toyota/Lexus/Subaru with TSK/ECU SECURITY KEY/SecOC
 
-![](https://user-images.githubusercontent.com/5363/91650158-ed5f5880-ea30-11ea-9b07-6e3dca7f8f83.gif)
-
-[^2]
+<figure>
+  <img src="https://user-images.githubusercontent.com/5363/91650158-ed5f5880-ea30-11ea-9b07-6e3dca7f8f83.gif" alt="RAV4 Prime CAN BUS traffic"/>
+  <figcaption>An image of the CAN BUS traffic on a RAV4 Prime. The "checksum" for the Lane Keep Assist messages are now very high in entropy, indicative of some sort of signing or encryption being used.</figcaption>
```

Comment:
This is rendering uglier than I hoped. Let me look for a better solution

Reply:
Gave up and moved it to postscriptAn image of the CAN BUS traffic on a RAV4 Prime. The "checksum" for the Lane Keep Assist messages are now very high in entropy, indicative of some sort of signing or encryption being used.